deployments/jwebbinar/image/environments/jwebbinar/build-hints.pip

```diff
@@ -1,2 +1,2 @@
-sphinx-asdf==0.1.3
-gwcs==0.18.0
\ No newline at end of file
+#sphinx-asdf
```

Comment:
Build hints are always "expendable",  but their purpose is to be carried over to chilly dependency resolution.   These are particularly strict or weird requirements w/o which chilly cannot find the overall solution since ~=x.y.z for these is not precise enough.  So #sphinx-asdf meaning"once used as a hint" makes sense,  but gwcs with no version should not help anything.

Reply:
Like nbconvert, this should go back in.  It was causing conflicts.